protoc-gen-debug/main.go

```diff
@@ -44,9 +44,13 @@ func main() {
 	}
 
 	// protoc-gen-debug supports proto3 field presence for testing purposes
-	var supportedFeatures = uint64(pluginpb.CodeGeneratorResponse_FEATURE_PROTO3_OPTIONAL)
+	var supportedFeatures = uint64(pluginpb.CodeGeneratorResponse_FEATURE_PROTO3_OPTIONAL) | uint64(pluginpb.CodeGeneratorResponse_FEATURE_SUPPORTS_EDITIONS)
+	var minimumEdition = int32(2023)
+	var maximumEdition = int32(2023)
```

Comment:
I believe this needs to use the editions constants from `google.golang.org/protobuf/types/descriptorpb`

```suggestion
	var minimumEdition = int32(descriptorpb.Edition_EDITION_2023)
	var maximumEdition = int32(descriptorpb.Edition_EDITION_2023)
```